src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -5,12 +5,37 @@ var LeafletPlainLayerView = require('./leaflet-plain-layer-view');
 var LeafletCartoDBLayerGroupView = require('./leaflet-cartodb-layer-group-view');
 var LeafletTorqueLayerView = require('./leaflet-torque-layer-view');
 var LeafletCartoDBWebglLayerGroupView = require('./leaflet-cartodb-webgl-layer-group-view');
+// Should we expose a Tangram.cartodb method for doing this?
+var CCSS = require('tangram-cartocss');
 
-var LayerGroupViewConstructor = function (layerGroupModel, mapModel, options) {
-  if (options.vector) {
-    return new LeafletCartoDBWebglLayerGroupView(layerGroupModel, mapModel);
+// Make this configurable and crate a pipe of decision based on the configuration.
+var shouldLoadWithVector = function (metadata, cb) {
+  // NOTE: the limit should be setted by the server?
+  if (metadata.stats && metadata.stats.size && metadata.stats.size > 10e6) {
+    return false;
   }
-  return new LeafletCartoDBLayerGroupView(layerGroupModel, mapModel);
+  else {
+    try {
+      metadata.layers.forEach(function (layer, i) {
+        if (layer.type === 'mapnik') {
+          CCSS.carto2Draw(layer.meta.cartocss, i);
+        }
+      });
+    }
+    catch (e) {
+      // TODO: show why it is not working and give feedback to the user
```

Comment:
I think this could be a good source of information to know what the properties we should implement are